cpp-linter/src/run.rs

```diff
@@ -21,14 +22,11 @@
 
 const VERSION: &str = env!("CARGO_PKG_VERSION");
 
-#[cfg(feature = "openssl-vendored")]
 fn probe_ssl_certs() {
+    #[cfg(feature = "openssl-vendored")]
     openssl_probe::init_ssl_cert_env_vars();
 }
```

Comment:
This function is unconditionally called in `run_main()`. I use the attribute in the function body to prevent duplicate function definitions depending on the conditional feature.